string map_id
---
bool success
string message
string[] obstacle_ids
Float1DArray[] lngs
Float1DArray[] lats
